pajbot/apiwrappers/twitch/tmi.py

```diff
@@ -0,0 +1,21 @@
+import logging
+
+from pajbot.apiwrappers.twitch.base import BaseTwitchAPI
+
+log = logging.getLogger(__name__)
+
+
+class TwitchTMIAPI(BaseTwitchAPI):
+    def __init__(self):
+        super().__init__(base_url="https://tmi.twitch.tv/")
+        self.session.headers["Accept"] = "application/vnd.twitchtv.v5+json"
```

Comment:
why this line? you don't need to send that header to the API at `tmi.twitch.tv`.